app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -315,6 +316,10 @@
         };
         return item.tier.toLowerCase() === (tierMap[predicate] || predicate);
       },
+      sublime: function(predicate, item) {
+        return item.isEngram() &&
+          item.name.indexOf('Sublime') === 0;
```

Comment:
Ah, makes sense.  Would handling this like the glimmer consumables make sense where there is a whitelist of item hashes?